PocketKit/Sources/PocketKit/Style/StyleConstants.swift

```diff
@@ -8,6 +8,10 @@ enum Margins: CGFloat {
   case wide = 32
 }
 
-enum StyleConstants {
+struct StyleConstants {
```

Comment:
I actually like having this be an enum since it mainly serves as a container for a few static properties. Making it a struct would allow creating instances of `StyleConstants` which could get confusing since it has no instance properties or methods.

Reply:
updated!